storage/statedb/database.go

```diff
@@ -677,6 +677,12 @@ func (db *Database) Dereference(root common.Hash) {
 	db.gcLock.Lock()
 	defer db.gcLock.Unlock()
 
+	// Sanity check to ensure that the meta-root is not removed
```

Comment:
This check can be moved before 677 line.

Reply:
@KimKyungup Moved those lines! https://github.com/klaytn/klaytn/pull/761/commits/ef40dc8d10ed209370c00d4f268cf76aba257b6a